### PR TITLE
jira ticket creation impovements

### PR DESCRIPTION
**Pull request by kobymeir**
## Status
- [x] Ready

## Description
fixing jira ticket creation:
 - attachment file link
 - sanitize the file name to not contain special chars
 - color the success/failed when reporting to jira
 - the test modeling rule didn't report success in the jira ticket

### CODE REVIEW

File: Tests/scripts/jira_issues.py [hunk 1]
```diff
@@ -53,6 +54,14 @@ def jira_file_link(file_name: str) -> str:
     return f"[^{file_name}]"
 
 
+def jira_sanitize_file_name(file_name: str) -> str:
+    return re.sub(r'[^\w-]+', '-', file_name).lower()
```
Comment (tkatzir) [inline Tests/scripts/jira_issues.py:58]: Couldn't it create conflicts with filenames like:
- "my file.txt" (1 space)
- "my  file.txt" (2 spaces)
Both will give my-file.txt
You can remediate by replacing every whitespace with a dash.

**Review by tkatzir — APPROVED** [review]
Great! Nice streamlining the colored messages.
Approved, with a minor comment.

File: Tests/scripts/jira_issues.py (hunk 1 above)
Comment (kobymeir) [reply Tests/scripts/jira_issues.py:58]: fixed

Comment (kobymeir) [inline Tests/scripts/jira_issues.py:58]:
```suggestion
    return re.sub(r'[^\w-]', '-', file_name).lower()
```

**Pull request merged**